ui/pages/onboarding-flow/recovery-phrase/review-recovery-phrase.js

```diff
@@ -158,13 +157,6 @@ export default function RecoveryPhrase({ secretRecoveryPhrase }) {
               >
                 {t('secretRecoveryPhrase')}
               </ButtonLink>,
-              <Text
-                key="seedPhraseReviewDetails2"
-                fontWeight={FontWeight.Medium}
-                color={TextColor.textAlternative}
-              >
-                {t('seedPhraseReviewDetails2')}
```

Comment:
the locale key was removed here but was not removed from messages.json files.. you can run `yarn verify-locales:fix` here. it will auto remove all unused locales listed